ui/app/src/views/projects/explore/ExploreView.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { useParams } from 'react-router-dom';
+import { DashboardResource, DEFAULT_DASHBOARD_DURATION, DEFAULT_REFRESH_INTERVAL } from '@perses-dev/core';
+import { generateMetadataName } from '../../../utils/metadata';
+import ProjectBreadcrumbs from '../../../components/ProjectBreadcrumbs';
+import HelperExploreView from './HelperExploreView';
+
+const DEFAULT_DASHBOARD_NAME = 'Explore Page';
+
+/**
+ * TBD
+ */
+function ExploreView() {
+  const { projectName } = useParams();
```

Comment:
Probably for a future PR: What about having this explore view also available on the home page / not linked to a project?

Reply:
this was something me and @Nexucis we discussed.
The idea of having it on the project level is to use to leverage global and project datasources.

I think might be confuse to the user Explore on two levels.
I mean the user cognitive load is a bit higher, because you need to think if you want to explore metrics from on project or from global source.

Reply:
Yeah valid point, still I think that if we make it explicit & with the same UX in both cases, this could work.

E.g we could imagine having a dropdown a the top to optionally select a project. Then, accessing the Explore view from your project space would prefill (+ lock) this dropdown, while accessing it from home page would leave it empty.

It could look like this:

![perses_proposal_explore_project_setting](https://github.com/perses/perses/assets/7058693/630a06d6-134d-43ac-b5e7-1fb097e149a6)

WDYT?

Reply:
This is a good idea btw!
Maybe we can simplify even more and have only one global explore page with a project drop down, wdyt?

Reply:
I agree with a single explore page with dropdown 😄